app/models/concerns/standardizable.rb

```diff
@@ -54,22 +54,26 @@ def override_setters
         when 'facility_code' then override_facility_code_setter
         when 'institution' then override_institution_setter
         when 'state' then override_state_setter
+        when 'ope' then override_ope_setter
+        when 'cross' then override_cross_setter
         else override_generic_setter(col, data_type)
         end
       end
     end
 
     def override_facility_code_setter
+      klass = self
+
       define_method 'facility_code=' do |value|
-        value = value.try(:strip)
-        self['facility_code'] = value ? value.upcase.rjust(8, '0') : nil
+        value = value.try(:strip).try(:upcase)
+        self['facility_code'] = klass.forbidden_word?(value) || value.blank? ? nil : value.rjust(8, '0')
```

Comment:
This doesn't seem right at all, you're using a method to dynamically define a new method? I'm not sure why this is necessary, but it's definitely a code smell.